staging/src/k8s.io/kubectl/pkg/cmd/create/create_deployment.go

```diff
@@ -53,7 +53,10 @@ var (
 	kubectl create deployment my-dep --image=nginx --replicas=3
 
 	# Create a deployment named my-dep that runs the busybox image and expose port 5701
-	kubectl create deployment my-dep --image=busybox --port=5701`))
+	kubectl create deployment my-dep --image=busybox --port=5701
+
+	# Create a deployment named my-dep that runs multiple containers
+	kubectl create deployment dep --image=busybox:latest --port=5701 --image=ubuntu:latest --image=nginx`))
```

Comment:
In this example, port number 5701 is only tighten to container whose image is busybox and not to others. To prevent possible misunderstandings, maybe we can remove port from here or mention this behavior. What do you think?.

Reply:
```suggestion
	kubectl create deployment my-dep --image=busybox:latest --image=ubuntu:latest --image=nginx`))
```

Reply:
> In this example, port number 5701 is only tied to container whose image is busybox and not to others. To prevent possible misunderstandings, maybe we can remove port from here or mention this behavior. What do you think?.

Hi @ardaguclu I am okay to do both , however, I'll prefer to remove the port so as it doesn't create confusion

Reply:
Since i have added the commit here , i realise the command for multiple containers can also be written as 
`kubectl create deployment my-dep --image=busybox,nginx` 

should i add this example too?